packages/@o3r/core/schematics/cms-adapter/index.ts

```diff
@@ -11,7 +12,7 @@ import * as commentJson from 'comment-json';
  * @param options.projectName
  * @param rootPath @see RuleFactory.rootPath
  */
-export function updateCmsAdapter(options: { projectName: string | null }, rootPath: string): Rule {
+export function updateCmsAdapter(options: CmsAdapterSchematicsSchema): Rule {
```

Comment:
Not from this PR, but I see that `options.projectName` is not used in this rule. I would suggest to pass it as the second argument when getting the workspaceProject (lines 55 and 106).